frameworks/Python/web2py/app/standard/modules/database.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+import os
+from operator import itemgetter
+from gluon.storage import Storage
+from gluon.dal import DAL, Field, Row
+
+DBHOST = os.environ.get('DBHOST', 'localhost')
+DATABASE_URI = 'mysql://benchmarkdbuser:benchmarkdbpass@%s:3306/hello_world' % DBHOST
+
+class Dal(object):
+    def __init__(self, table=None, pool_size=8):
+        self.db = DAL(DATABASE_URI, migrate_enabled=False, pool_size=pool_size)
+        if table == 'World':
+            self.db.define_table('World', Field('randomNumber', 'integer'))
+        elif table == 'Fortune':
+            self.db.define_table('Fortune', Field('message'))
+
+    def get_world(self, wid):
+        return self.db(self.db.World.id == wid).select(cacheable=True)[0].as_dict()
```

Comment:
No, `cacheable=True` simply tells the DAL to return a `Rows` object that *could be* cached. The only reason for setting that to `True` here is that in addition to allowing the results to be cached (with this code is *not* doing), it also speeds things up a bit because it foregoes creation of some additional attributes.